tools/packaging/Arch_User_Repository/PKGBUILD

```
@@ -8,7 +8,7 @@ arch=('x86_64')
 url="https://www.precice.org"
 license=('LGPL3')
 depends=('openmpi' 'boost')
-makedepends=('cmake')
+makedepends=('cmake', 'eigen')
```

This is actually not valid, it needs to be:
```suggestion
makedepends=('cmake' 'eigen')
```

---

Other than that, it's ok, I have tested it in Manjaro.